### PR TITLE
run Grafana as non root

### PR DESCRIPTION
**Pull request by tombart**
#### What this PR does / why we need it:
Run grafana as non root user, due to security issues it is not recommended to run this tool as root user.
#### Which issue this PR fixes
uses grafana user to run grafana

#### Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [ ] [DCO](https://www.helm.sh/blog/helm-dco/index.html) signed
- [ ] Chart Version bumped
- [ ] Variables are documented in the README.md

### CODE REVIEW

File: stable/grafana/templates/deployment.yaml [hunk 1]
```diff
@@ -59,6 +59,15 @@ spec:
               readOnly: {{ .readOnly }}
           {{- end }}
 {{- end }}
+        - name: set-permissions
+          image: busybox
+          command: ["/bin/sh", "-c", "mkdir -m 777 /var/lib/grafana/plugins && chmod -R 777 /var/lib/grafana/"]
```
Comment (mochi99999) [inline stable/grafana/templates/deployment.yaml]: Should we run chown 472:472 instead of chmod 777?

Comment (tombart) [reply stable/grafana/templates/deployment.yaml]: yes, you're right, no need to set 777, just changed that, thanks

**Comment by tombart** [comment]
Hi guys, are we ok to merge this? thanks

**Comment by tombart** [comment]
Hi, is it ok to merge this change guys?

**Pull request closed without merging**